docs/modules/reference/pages/assemble/jlink.adoc

```diff
@@ -463,7 +463,7 @@ include::partial$assemble/maven/common-head.adoc[]
           <multiRelease>base</multiRelease>
 
           <!--
-            Ignore missing dependences.
+            Ignore missing dependencies.
```

Comment:
This is actually not a typo. I thoght so too as well but jdeps insists in called them "dependences" 🤷‍♂️

Reply:
Ok :) That's an interesting choice.